core/tests/gauge-taiko automated tests/voiceover-tests/package-lock.json

```diff
@@ -0,0 +1,1242 @@
+{
```

Comment:
Why do we need an additional package here? We would prefer not to deal with nested repositories, is this actually needed or can it be part of the standard package.json at oppia's root folder?